android/app/build.gradle

```diff
@@ -98,6 +98,7 @@ dependencies {
     implementation 'com.github.jasminb:jsonapi-converter:0.7'
     implementation 'com.squareup.retrofit2:retrofit:2.3.0'
     implementation 'com.squareup.retrofit2:converter-jackson:2.3.0'
+    compile 'com.fasterxml.jackson.module:jackson-module-kotlin:2.9.+'
```

Comment:
Just adding the dependency does not work. You'll have to register it in the object mapper